tasks/src/main/scala/dagr/tasks/picard/MergeBamAlignment.scala

```diff
@@ -42,11 +42,13 @@ class MergeBamAlignment(unmapped: PathToBam,
                         programGroupCommand: Option[String] = None,
                         programGroupName: Option[String] = None,
                         attributesToRetain: List[String] = List[String]("X0", "ZS", "ZI", "ZM", "ZC", "ZN"),
+                        attributesToReverse: List[String] = List[String]("ad", "bd", "cd", "ae", "be", "ce"), // fgbio consensus tags
```

Comment:
Once we get the duplex caller done they'll all be in the `ConsensusTags` object.  Just not quite there yet and didn't want to have to make a second PR here later.